video_creation/screenshot_downloader.py

```diff
@@ -105,7 +120,21 @@ def get_screenshots_of_reddit_posts(reddit_object: dict, screenshot_num: int):
             print_substep("Skipping translation...")
 
         postcontentpath = f"assets/temp/{reddit_id}/png/title.png"
-        page.locator('[data-test-id="post-content"]').screenshot(path=postcontentpath)
+        try:
+            page.locator('[data-test-id="post-content"]').screenshot(path=postcontentpath)
+        except Exception as e:
+            print_substep("Something went wrong!",style="red")
+            resp = input("Something went wrong with making the screenshots! Do you want to skip the post? (y/n) ")
+
+            if resp.casefold().startswith("y"):
+                save_data("", "", "skipped", reddit_id, "")
+                print_substep("The post is successfully skipped! You can now restart the program and this post will skipped.","green")
+
+            resp = input("Do you want the error traceback for debugging purposes? (y/n)")
+            if not resp.casefold().startswith("y"):
+                exit()
+            
+            raise e
```

Comment:
Really lazy implementation instead of just fixing the problem

Reply:
The problem is fixed, but it can always go wrong so we give the user the option to skip